Source/CombatExtended/CombatExtended/Jobs/JobGiver_TakeAndEquip.cs

```diff
@@ -81,7 +81,8 @@ private WorkPriority GetPriorityWork(Pawn pawn)
                 
             	if (hasWeaponTags
                   && primaryAmmoUser.parent.def.weaponTags.Any(pawn.kindDef.weaponTags.Contains)
-            	  && loadoutPropertiesExtension?.primaryMagazineCount != FloatRange.Zero)
+                  && loadoutPropertiesExtension != null
+                  && loadoutPropertiesExtension?.primaryMagazineCount != FloatRange.Zero)
```

Comment:
Why check for null twice?

Reply:
I think you mean the ? on the line after the null check. Removed.